### DISCUSSION TITLE
Airbyte does not load, fails for pgdata

### DISCUSSION THREAD

**Discussion by siemrbg** [airbytehq/airbyte#40691] [Platform General Questions]
I am using Windows version for abctl and have Docker Desktop up and running on Windows.

PS C:\Users\> abctl local install
  INFO    Thanks for using Airbyte!
          Anonymous usage reporting is currently enabled. For more information, please see https://docs.airbyte.com/telemetry
  INFO    Using Kubernetes provider:
            Provider: kind
            Kubeconfig: C:\Users\.airbyte\abctl\abctl.kubeconfig
            Context: kind-airbyte-abctl
SUCCESS  Found Docker installation: version 26.1.4
SUCCESS  Port 8000 appears to be available
  INFO    No existing cluster found, cluster 'airbyte-abctl' will be created
SUCCESS  Cluster 'airbyte-abctl' created
  INFO    Namespace 'airbyte-abctl' created
  INFO    Persistent volume 'airbyte-minio-pv' created
  INFO    Persistent volume 'airbyte-volume-db' created
  INFO    Persistent volume claim 'airbyte-minio-pv-claim-airbyte-minio-0' created
  INFO    Persistent volume claim 'airbyte-volume-db-airbyte-db-0' created
WARNING  Encountered an issue deploying Airbyte:
            Pod: airbyte-db-0.17de7a9772711da8
            Reason: BackOff
            Message: Back-off restarting failed container airbyte-db-container in pod airbyte-db-0_airbyte-abctl(a110a01b-7bbc-4c5f-a873-9ec1eb11f9a4)
            Count: 6
            Logs: mkdir: can't create directory '/var/lib/postgresql/data/pgdata': Permission denied

What settings do I need to do to resolve the permission error ?

**Comment by johnwasserman** [comment]
Hi @siemrbg - thanks for highlighting this!

We should have a fix for this issue in this [Pull Request](https://github.com/airbytehq/abctl/pull/43).  In summary, it is a permissions issue when docker is running as root/admin and the abctl script is running as a non-admin/root user.

The next release of abctl (going out sometime this week) will include this fix.

[a comment by an automated account is not shown]
